backend/src/controllers/ScheduleController.ts

```diff
@@ -45,27 +45,30 @@ class ScheduleController {
     }
   }
 
-  // DELETE an existing schedule using JWT
-  public static async deleteScheduleByID(req: any, res: any) {
+  // DELETE all time slots in a schedule using JWT
+  public static async clearScheduleById(req: any, res: any) {
```

Comment:
Updated this endpoint so that sets the `timeSlot` array into an empty array instead of deleting the schedule completely, which was causing an error. Changed the name from `deleteScheduleByID` to `clearScheduleById` for semantic purposes.